### ISSUE TITLE
[REF-1065] Reflex init __ UnicodeDecodeError: 'utf-8' codec can't decode byte 0x82 in position 8: invalid start byte

### ISSUE THREAD

**Issue by davicillo78** [reflex-dev/reflex#1952]
```console
(venv) PS C:\WEB_PY_REFLEX\proyecto1> reflex init
Info: Overriding config value username with env var USERNAME=David
─────────────────────────────────────────────── Initializing proyecto1 ────────────────────────────────────────────────
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\WEB_PY_REFLEX\venv\Scripts\reflex.exe\__main__.py", line 7, in <module>
  File "C:\WEB_PY_REFLEX\venv\Lib\site-packages\typer\main.py", line 214, in __call__
    return get_command(self)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\WEB_PY_REFLEX\venv\Lib\site-packages\click\core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\WEB_PY_REFLEX\venv\Lib\site-packages\click\core.py", line 1078, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "C:\WEB_PY_REFLEX\venv\Lib\site-packages\click\core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\WEB_PY_REFLEX\venv\Lib\site-packages\click\core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\WEB_PY_REFLEX\venv\Lib\site-packages\click\core.py", line 783, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\WEB_PY_REFLEX\venv\Lib\site-packages\typer\main.py", line 532, in wrapper
    return callback(**use_params)  # type: ignore
           ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\WEB_PY_REFLEX\venv\Lib\site-packages\reflex\reflex.py", line 76, in init
    prerequisites.initialize_frontend_dependencies()
  File "C:\WEB_PY_REFLEX\venv\Lib\site-packages\reflex\utils\prerequisites.py", line 606, in initialize_frontend_dependencies
    processes.run_concurrently(install_node, install_bun)
  File "C:\WEB_PY_REFLEX\venv\Lib\site-packages\reflex\utils\processes.py", line 192, in run_concurrently
    with run_concurrently_context(*fns):
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.12_3.12.240.0_x64__qbz5n2kfra8p0\Lib\contextlib.py", line 144, in __exit__
    next(self.gen)
  File "C:\WEB_PY_REFLEX\venv\Lib\site-packages\reflex\utils\processes.py", line 183, in run_concurrently_context
    task.result()
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.12_3.12.240.0_x64__qbz5n2kfra8p0\Lib\concurrent\futures\_base.py", line 449, in result
    return self.__get_result()
           ^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.12_3.12.240.0_x64__qbz5n2kfra8p0\Lib\concurrent\futures\_base.py", line 401, in __get_result
    raise self._exception
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.12_3.12.240.0_x64__qbz5n2kfra8p0\Lib\concurrent\futures\thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\WEB_PY_REFLEX\venv\Lib\site-packages\reflex\utils\prerequisites.py", line 414, in install_node
    processes.show_status("Installing node", process)
  File "C:\WEB_PY_REFLEX\venv\Lib\site-packages\reflex\utils\processes.py", line 248, in show_status
    for line in stream_logs(message, process):
  File "C:\WEB_PY_REFLEX\venv\Lib\site-packages\reflex\utils\processes.py", line 215, in stream_logs
    for line in process.stdout:
  File "<frozen codecs>", line 322, in decode
UnicodeDecodeError: 'utf-8' codec can't decode byte 0x82 in position 8: invalid start byte
```

Specifics (please complete the following information):

Python Version: 3.12
Reflex Version: Reflex v0.2.9
OS: Windows 10
Browser (Optional): Chrome

<sub>[REF-1065](https://linear.app/reflex-dev/issue/REF-1065/reflex-init-unicodedecodeerror-utf-8-codec-cant-decode-byte-0x82-in)</sub>

**Comment by Delacox** [comment]
I'm the same, did you solve the problem?

**Comment by picklelo** [comment]
Are you using Python 3.12? We should have a release in the upcoming weeks with full 3.12 support, currently there may be some bugs.

**Comment by masenf** [comment]
I think at least one potential problem here is that Reflex assumes the target system encoding is "utf-8", which is not generally a safe assumption on windows. If `subprocess` is trying to decode command output as utf-8 but it's actually cp1252 or something else, then we're going to hit these decode errors...

i'm wondering if the solution here is to ignore the encoding and read+write raw bytes to the terminal. Then, regardless of the system encoding, we don't really have to deal with it... in the cases where Reflex needs to inspect the output streams, it can try to decode them (or just use byte regex).

**Comment by NakanoSanku** [comment]
I'm the same,
- python version: 3.10.13
- powershell :5.1.22621.2506 
- reflex version:Version: 0.3.4
- OS: windows 11 23H2
```
(c:\Users\KateT\Desktop\Autoyys_python\.conda) PS C:\Users\KateT\Desktop\Autoyys_python\web> reflex init
─────────────────────────────────────────────────── Initializing web ───────────────────────────────────────────────────
Traceback (most recent call last):
  File "c:\Users\KateT\Desktop\Autoyys_python\.conda\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\Users\KateT\Desktop\Autoyys_python\.conda\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "c:\Users\KateT\Desktop\Autoyys_python\.conda\Scripts\reflex.exe\__main__.py", line 7, in <module>
  File "c:\Users\KateT\Desktop\Autoyys_python\.conda\lib\site-packages\typer\main.py", line 328, in __call__
    raise e
  File "c:\Users\KateT\Desktop\Autoyys_python\.conda\lib\site-packages\typer\main.py", line 311, in __call__
    return get_command(self)(*args, **kwargs)
  File "c:\Users\KateT\Desktop\Autoyys_python\.conda\lib\site-packages\click\core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
  File "c:\Users\KateT\Desktop\Autoyys_python\.conda\lib\site-packages\typer\core.py", line 778, in main
    return _main(
  File "c:\Users\KateT\Desktop\Autoyys_python\.conda\lib\site-packages\typer\core.py", line 216, in _main
    rv = self.invoke(ctx)
  File "c:\Users\KateT\Desktop\Autoyys_python\.conda\lib\site-packages\click\core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "c:\Users\KateT\Desktop\Autoyys_python\.conda\lib\site-packages\click\core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "c:\Users\KateT\Desktop\Autoyys_python\.conda\lib\site-packages\click\core.py", line 783, in invoke
    return __callback(*args, **kwargs)
  File "c:\Users\KateT\Desktop\Autoyys_python\.conda\lib\site-packages\typer\main.py", line 683, in wrapper
    return callback(**use_params)  # type: ignore
  File "c:\Users\KateT\Desktop\Autoyys_python\.conda\lib\site-packages\reflex\reflex.py", line 134, in init
    _init(name, template, loglevel)
  File "c:\Users\KateT\Desktop\Autoyys_python\.conda\lib\site-packages\reflex\reflex.py", line 95, in _init
    prerequisites.initialize_frontend_dependencies()
  File "c:\Users\KateT\Desktop\Autoyys_python\.conda\lib\site-packages\reflex\utils\prerequisites.py", line 667, in initialize_frontend_dependencies
    processes.run_concurrently(install_node, install_bun)
  File "c:\Users\KateT\Desktop\Autoyys_python\.conda\lib\site-packages\reflex\utils\processes.py", line 192, in run_concurrently
    with run_concurrently_context(*fns):
  File "c:\Users\KateT\Desktop\Autoyys_python\.conda\lib\contextlib.py", line 142, in __exit__
    next(self.gen)
  File "c:\Users\KateT\Desktop\Autoyys_python\.conda\lib\site-packages\reflex\utils\processes.py", line 183, in run_concurrently_context
    task.result()
  File "c:\Users\KateT\Desktop\Autoyys_python\.conda\lib\concurrent\futures\_base.py", line 451, in result
    return self.__get_result()
  File "c:\Users\KateT\Desktop\Autoyys_python\.conda\lib\concurrent\futures\_base.py", line 403, in __get_result
    raise self._exception
  File "c:\Users\KateT\Desktop\Autoyys_python\.conda\lib\concurrent\futures\thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "c:\Users\KateT\Desktop\Autoyys_python\.conda\lib\site-packages\reflex\utils\prerequisites.py", line 475, in install_node
    processes.show_status("Installing node", process)
  File "c:\Users\KateT\Desktop\Autoyys_python\.conda\lib\site-packages\reflex\utils\processes.py", line 249, in show_status
    for line in stream_logs(message, process):
  File "c:\Users\KateT\Desktop\Autoyys_python\.conda\lib\site-packages\reflex\utils\processes.py", line 216, in stream_logs
    for line in process.stdout:
  File "c:\Users\KateT\Desktop\Autoyys_python\.conda\lib\codecs.py", line 322, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xcb in position 0: invalid continuation byte
```

**Comment by masenf** [comment]
Fixed in 0.4.9
